pkg/pillar/cmd/volumemgr/updatestatus.go

```diff
@@ -282,16 +282,23 @@ func doUpdateContentTree(ctx *volumemgrContext, status *types.ContentTreeStatus)
 			return changed, false
 		}
 		for _, loadedBlob := range loadedBlobs {
-			log.Infof("doUpdateContentTree(%s): Successfully loaded blob: %s", status.Key(), loadedBlob.Sha256)
-			if loadedBlob.State == types.LOADED && loadedBlob.HasVerifierRef {
+			// cas client take a time, so we need to lookup new status of blob
+			refreshedBlob := lookupBlobStatus(ctx, loadedBlob.Sha256)
+			if refreshedBlob == nil {
+				refreshedBlob = loadedBlob // use new one if not exists
```

Comment:
That is very suspect. If it doesn't exist that means it disappeared while ingestion happened, and in that case you have nothing to do